jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -76,7 +79,7 @@ def process(self, input):
                     message = "MSA metadata file (MSAMETFL) is required for NRS_MSASPEC exposures."
                     log.error(message)
                     raise MissingMSAFileError(message)
-
-            result = load_wcs(input_model, reference_file_names)
+            slit_y_range = self.slit_y_low, self.slit_y_high
```

Comment:
You refer to this later as a list.  If it matters that it should be a list rather than a tuple, you should enclose the terms on the right hand side in brackets.

Reply:
Any iterable is fine at this point. I enclosed it in brackets for clarity.